packages/rolldown/tests/fixture.test.ts

```diff
@@ -1,33 +1,25 @@
+/// <reference types="vite/client" />
```

Comment:
Feel this not ideal. Maybe vitest could exposed these from https://github.com/vitest-dev/vitest/blob/main/packages/vitest/importMeta.d.ts.

Reply:
Yes, I will report this issue to `vitest`